### PR TITLE
Add postgresfm podcast

### PR DESCRIPTION
**Pull request by gulcin**
## Summary

-

### Blog Release Checklist

- \[ ] Release on: \<mm/dd/yy>
- \[ ] Published on dev.to (connected to Xata org)
- \[ ] OG image included
- \[ ] Social media is scheduled for this post

### CODE REVIEW

File: postgresfm.mdx [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+## Podcast Appearance: Postgres.fm 🎙
+
+I was invited to the [Postgres.fm](https://postgres.fm) podcast together with [Robert Haas](https://www.linkedin.com/in/robertmhaas/), hosted by [Michael Christofides](https://postgres.fm/people/michael-christofides) and [Nikolay Samokhvalov](https://postgres.fm/people/nikolay-samokhvalov), to discuss the technical aspects of the question: \_Is pg_dump a backup tool, and in what way is it or is not?
+The podcast experience was much better for me this time, as I was joined by Robert, compared to solo episodes I’ve done before.
+Robert and I had already been exploring this topic through our blogs, and it became a topic we liked to chat about.
+
+That familiarity carried over into the podcast, making the discussion flow naturally.
+We had a great time recording this episode and hope listeners enjoy it too!
+
+[![Is pg_dump a backup tool?](/images/postgresfm/postgres-fm-youtube)](https://youtu.be/-DO5ad-KtFY)
```
Comment (tsg) [inline postgresfm.mdx:54]: Missing `.png` here?

Comment (gulcin) [reply postgresfm.mdx:54]: Ah, yes. Thanks @tsg

**Review by alexfrancoeur — APPROVED** [review]
I messed up your italics but I'm pretty sure I just fixed it. LGTM ✅

**Comment by gulcin** [comment]
> I messed up your italics but I'm pretty sure I just fixed it. LGTM ✅

Thanks @alexfrancoeur 🎉

**Pull request merged**